### ISSUE TITLE
'lerna run' fails after upgrade to v6.4.0

### ISSUE THREAD

**Issue by donmccurdy** [lerna/lerna#3491]
## Current Behavior

Running `lerna run dist` fails without a meaningful error message in v6.4.0, while running the `dist` command in any individual package (e.g. `yarn dist`) succeeds. Execution succeeded in v6.3.0. If I revert to `"useNx": false,` the command succeeds in v6.4.0.

## Expected Behavior

`lerna run dist` should execute the command in all workspaces. This appears to be a regression in v6.4.

## Steps to Reproduce

I wasn't able to reproduce the problem in the 'repro' repository, but hopefully this is enough:

1. Clone https://github.com/donmccurdy/glTF-Transform
2. Run `yarn install`
3. Run `yarn dist`, or equivalently `npx lerna run dist` ✅
4. Upgrade Lerna to v6.4.0
5. Run `yarn dist`, or equivalently `npx lerna run dist` ❌

If I set `"useNx": false`, everything works.

### Failure Logs / Configuration

The log warning "lerna verb run Nx target configuration was not found. Task dependencies will not be automatically included." seems suspicious. I'm not aware that 6.4 should require any new configuration as compared to 6.3, so I'm not sure what to do with this warning.

<details><summary>output</summary><p>

```text
yarn dist --verbose --loglevel debug
yarn run v1.22.19
$ lerna run dist --verbose --loglevel debug
lerna notice cli v6.4.0
lerna verb rootPath /Users/donmccurdy/Documents/Projects/glTF-Transform
lerna verb run Nx target configuration was not found. Task dependencies will not be automatically included.

 >  Lerna (powered by Nx)   Running target dist for 4 project(s):

    - @gltf-transform/cli
    - @gltf-transform/core
    - @gltf-transform/extensions
    - @gltf-transform/functions

 ——————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————

> @gltf-transform/core:dist


 ——————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————

 >  Lerna (powered by Nx)   Running target "dist" failed

   Tasks not run because their dependencies failed or --nx-bail=true:

   - @gltf-transform/cli:dist
   - @gltf-transform/extensions:dist
   - @gltf-transform/functions:dist

   Failed tasks:

   - @gltf-transform/core:dist

error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

</details>

<details><summary>lerna.json</summary><p>

```json
{
	"version": "3.0.0-alpha.3",
	"npmClient": "yarn",
	"useWorkspaces": true,
	"packages": ["packages/*"]
}
```

</p></details>

### Environment

 Environment info:

  System:
    OS: macOS 13.1
    CPU: (8) arm64 Apple M1 Pro
  Binaries:
    Node: 16.18.1 - ~/.nvm/versions/node/v16.18.1/bin/node
    Yarn: 1.22.19 - ~/.nvm/versions/node/v16.18.1/bin/yarn
    npm: 8.19.2 - ~/.nvm/versions/node/v16.18.1/bin/npm
  Utilities:
    Git: 2.37.1 - /usr/bin/git
  npmPackages:
    lerna: ^6.4.0 => 6.4.0

**Comment by itsikcircle** [comment]
+1 having the same issue

**Comment by blvdmitry** [comment]
+1, same behaviour. I went from 5.x to 6.4 directly and the only error I get is:

```
Lerna (powered by Nx)   Ran target undefined for project reshaped
```

when running

```
yarn lerna run build --scope reshaped
```

**Comment by JamesHenry** [comment]
Hi Folks, thanks again for reporting this, it should hopefully be resolved in lerna 6.4.1 which was just released

**Comment by blvdmitry** [comment]
Worked for me, thanks!

**Comment by donmccurdy** [comment]
Confirmed fixed in v6.4.1, thank you!
